skrebate/surf.py

```diff
@@ -27,8 +27,9 @@
 from __future__ import print_function
 import numpy as np
 from sklearn.externals.joblib import Parallel, delayed
-from .relieff import ReliefF
-from .scoring_utils import SURF_compute_scores
+from relieff import ReliefF
+from scoring_utils import SURF_compute_scores
```

Comment:
Just a quick look. Please edit two lines as below for passing unit tests. 
```
from .relieff import ReliefF
from .scoring_utils import SURF_compute_scores
```
You need the `.` for importing.